utils/build-script

```diff
@@ -701,6 +701,13 @@ the number of parallel build jobs to use""",
         action="append", dest="extra_swift_args", default=[])
 
     parser.add_argument(
+        "--extra-cmake-vars", help=textwrap.dedent("""
```

Comment:
We call them 'options' throughout the build-script, so it would me more consistent to call it `--extra-cmake-options`